4458 - Add clearable, autoselect, text-align and update ids-input (#21)
infor-design/enterprise-wc: src/ids-input/README.md

<ids-input label="Last Name" validate="required"></ids-input>
```

> Set the Clearable to Text Input this way:

Reviewer: ```suggestion
Set the input to clearbale and add a clear button:
```
